.jenkins/common_job_properties.groovy

```diff
@@ -86,14 +86,19 @@ class common_job_properties {
   // Sets the pull request build trigger.
   static def setPullRequestBuildTrigger(def context,
                                         def commitStatusContext,
-                                        def successComment = '--none--') {
+                                        def successComment = '--none--',
```

Comment:
I've made some changes in the other PR which should address this -- once we settle on those and that gets committed I'll pull them into here too.